library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942_20cu/Equality/aquavanjaram_Cijk_Ailk_Bjlk_HHS_BH_UserArgs.yaml

```diff
@@ -82,4 +82,4 @@
 - null
 - null
 - DeviceEfficiency
-- GridBased
+- Equality
```

Comment:
I am not sure how you generated this yaml, but only this file has 3 "- null".
It should be 2.
I just realized this does not have any size information.
Then, we do not need this file.